jax_unirep/featurize.py

```diff
@@ -30,17 +29,14 @@ def rep_same_lengths(
 
     embedded_seqs = get_embeddings(seqs)
 
-    init_fun, apply_fun = mLSTM1900()
-    _, params = init_fun(PRNGKey(0), (-1, 10))
```

Comment:
Hold on, I think the amount of code duplication here can be minimized. Only line 34 needs to be deleted, line 33 can be kept, and possibly could be changed to:

```python
_, apply_fun = mLSTM1900()
```

Alternatively, as we discussed in Teams, we can also move the instantiation to the module level, to reduce the number of function calls.

Doing so will minimize the number of arguments to `rep_same_lengths`, or phrased in a more abstract sense, the surface area of the function.

With this smaller change, the number of line changes below can also be minimized.